adding 2 commands, MaxConcurrentQueries and MaxDelayQueueSize (#319)
tigergraph/server-docs: modules/reference/pages/configuration-parameters.adoc

> |RESTPP.WorkLoadManager.MaxHeavyBuiltinQueries |The maximum concurrent
> running “heavy“ built-in queries(kstep_expansion, searchvertex,

Reviewer: ```suggestion
|RESTPP.WorkLoadManager.MaxHeavyBuiltinQueries |The maximum number of concurrent
“heavy“ built-in queries (kstep_expansion, searchvertex,
```

allpaths, shortestpath) on a restpp server |`100`

> |RESTPP.WorkLoadManager.MaxConcurrentQueries |The maximum concurrent

Reviewer: ```suggestion
|RESTPP.WorkLoadManager.MaxConcurrentQueries |The maximum number of concurrent
```

> queries allowed to be running, *does not include* heavy queries | `50`

Reviewer: ```suggestion
queries allowed to run, *excluding* heavy queries | `50`
```
